jdi-light-examples/src/test/java/io/github/epam/tests/nonstatic/NonStaticSiteTests.java

```diff
@@ -0,0 +1,19 @@
+package io.github.epam.tests.nonstatic;
+
+import io.github.epam.NonStaticTestsInit;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+public class NonStaticSiteTests extends NonStaticTestsInit {
+
+    @BeforeMethod
+    public void before() {
+        nonStaticSite.shouldBeLoggedIn();
+        nonStaticSite.getHomePage().shouldBeOpened();
+    }
+
+    @Test
+    public void isOpenedTest() {
+        nonStaticSite.getHomePage().isOpened();
+    }
+}
```

Comment:
EOF should be at the empty line. Fix it, please

Reply:
ok done